src/index.ts

```diff
@@ -121,6 +121,7 @@ export class SerialPort {
 }
 
 export class FileSystem {
```

Comment:
Add comment above function using /** */. 
In this way, comment of function will appear when user hovers over function name.

Reply:
![image](https://user-images.githubusercontent.com/41713563/65740020-ec83bf80-e119-11e9-9c48-321801bd59d7.png)

Reply:
Type `/**` and enter, VS Code will scaffold the structure for you.

Reply:
Fixed.